readable.js

```diff
@@ -1,6 +1,9 @@
-var Stream = require('stream'); // hack to fix a circular dependency issue when used with browserify
+(function (){
+  try {
+    exports.Stream = require('st' + 'ream'); // hack to fix a circular dependency issue when used with browserify
```

Comment:
what happens here? could this ever fail? (excuse me if I'm asking dumb questions)